code/obj/window.dm

```diff
@@ -26,7 +26,7 @@
 	var/default_material = "glass"
 	var/default_reinforcement = null
 	var/reinf = 0 // cant figure out how to remove this without the map crying aaaaa - ISN
-	var/deconstruct_time = 0//20
+	var/deconstruct_time = 10
```

Comment:
10 deciseconds? 1 second?

Reply:
All spots that set deconstruct time now use SECOND/S for clarity.